lib/components/Dropdown/index.tsx

```diff
@@ -29,6 +29,7 @@ type TProps = HtmlHTMLAttributes<HTMLInputElement> & {
     name: InputProps["name"];
     onSearch?: (inputValue: string) => void;
     onSelect: (value: string) => void;
+    selectedVal?: InputProps["value"];
```

Comment:
```suggestion
    selectedItem?: InputProps["value"];
```
or `selecteValue` ? 🤔